examples/oneapi/data/svm_reg_test_dense_label.csv

```diff
@@ -148,103 +148,3 @@
 -554.87
 -2344.25
 -93.33
-735.76
```

Comment:
What is the reason to cut this files?

Reply:
Idea was that test data should be proportionally reduced - like with SVM we have 500 rows for training and it make little sense to have large amount for testing. But probably we can keep them as is